go.mod

```diff
@@ -1,5 +1,7 @@
 module kubevirt.io/kubevirt
 
+go 1.12
```

Comment:
i think we need this line to be removed

Reply:
why?  vscode inserts it automatically

Reply:
i believe it's set dependent on your local environment. it seems to change all the time each time someone changes a dependency, and doesn't necessarily reflect what it's meant to reflect.

Reply:
1.12 is the correct value assuming people use `make deps-update` if people are using their local environment to update dependencies, then it is possible that their local go version is newer and will make a modules.txt that is in the wrong format.  Basically, this value should ideally be there and match the go version we using.

Reply:
not everyone is using bazel or containerized builds.

Reply:
leaving this out increases the likelihood that there will be a mismatch between modules.txt format and whatever compiler is being used.  I fail to see how leaving it out makes anything better.  Newer compilers can read/write older modules.txt format.  But since this PR has been open for 2+ months I will remove it if that will help to finally get this merged.

Reply:
I'm sure you're right. I just wouldn't introduce that in this pr. I honestly don't completely understand the implications of this line.